.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Awesome!!!! 🚀 thanks for that, I wil try to implement in our project. Can you recommend some learning sources?